pxd_fastpath_stub.c

```diff
@@ -24,4 +23,28 @@ int pxd_init_fastpath_target(struct pxd_device *pxd_dev, struct pxd_update_path_
 	return 0; // cannot fail
 }
 
+void pxd_fastpath_adjust_limits(struct pxd_device *pxd_dev, struct request_queue *topque) {}
+int pxd_suspend_state(struct pxd_device *pxd_dev) {return 0;}
+
+void pxd_suspend_io(struct pxd_device* pxd_dev) { }
+void pxd_resume_io(struct pxd_device* pxd_dev) { }
+int pxd_switch_fastpath(struct pxd_device* pxd_dev) {return -1;}
+int pxd_switch_nativepath(struct pxd_device* pxd_dev) {return -1;}
```

Comment:
traces for these functions where we dont expect them to be called? I would have said ASSERTs but I am not sure if a kernel panic is what we want. Is there any other way to ensure these functions are not getting called, perhaps a test only module which will have asserts here.

Reply:
it's not that they can't be called, we are not going to advertise fastpath.